docs/ref/clvm.md

```diff
@@ -742,7 +745,249 @@ cost = 806
+                        'amount': 1000000000000,
+                        'parent_coin_info': '0x98706c3a489574a9f32f339662f339b7ba3c9dce1da4feb301432aa60216635e',
+                        'puzzle_hash': '0xd8af3cb1130f6d7e4011c6fa85779c0cfddb1a594cdd170d1dfc8aeb5f3c93fe'
+                     },
+                        'puzzle_reveal': '0xff02ffff01ff02ffff01ff02ffff03ff0bffff01ff02ffff03ffff09ff05ffff1dff0bffff1effff0bff0bffff02ff06ffff04ff02ffff04ff17ff8080808080808080ffff01ff02ff17ff2f80ffff01ff088080ff0180ffff01ff04ffff04ff04ffff04ff05ffff04ffff02ff06ffff04ff02ffff04ff17ff80808080ff80808080ffff02ff17ff2f808080ff0180ffff04ffff01ff32ff02ffff03ffff07ff0580ffff01ff0bffff0102ffff02ff06ffff04ff02ffff04ff09ff80808080ffff02ff06ffff04ff02ffff04ff0dff8080808080ffff01ff0bffff0101ff058080ff0180ff018080ffff04ffff01b0848f09f98800442737684dd76071f25a0bd100b51e727aabafeddb062dbc3d2b3ac64bc87f084a6d16e4e89e1417de14ff018080',
+                        'solution': '0xff80ffff01ffff3dffa023f61666150d2a467ee7b81a77954c93255d65c0c43108f1bb14ac420fd59c428080ff8080'
+                  }]},
+}
+```
+
+The `additions` and `removals` sections contain two coins apiece. Before the transaction, Alice had two coins in her wallet, one worth 1 XCH, and one worth 0.946839958396 XCH. The amount sent was 1.01 XCH, which was greater than the value of either of Alice's coins. The wallet therefore automatically selected both coins to be sent, for a total of 1.946839958396 XCH. A new coin worth 0.936839958396 XCH was created in Alice's wallet as "change". 
+
+To recap, before the transaction:
+* Alice had two coins, worth 1 XCH and 0.946839958396 XCH
+* Bob had zero coins
+* The total value was 1.946839958396 XCH
+
+After the transaction:
+* Alice had one coin worth 0.936839958396 XCH
+* Bob had one coin worth 1.01 XCH
+* The total value was 1.946839958396 XCH (same as before the transaction)
+
+Back to the original question: what was the CLVM cost of this transaction? One way to find out is by creating a spend bundle based on the output of the `get_transaction` command, and inspecting it using `cdv`.
+
+  > NOTE: You'll need to run `pip install chia-dev-tools` in order to use the `cdv` command.
+
+```bash 
+(venv) chia-blockchain $ cdv inspect spendbundles spend.json -ec
+...
+Cost: 17187295
+```
+
+The cost was 17,187,295. Note that this command is simulating a small block that contains only the single spendbundle. In reality, this spendbundle would be aggregated into a larger block, possibly with 999 similar transactions. Therefore, this cost is only an estimation. However, for many applications, this simple technique will be sufficient.
+
+As for fee calculations, the minimum transaction fee is 5 mojos per cost (anything less than this will be treated as 0). In this case, the minimum fee is `5 * 17,187,295 = 85,936,475 mojos`. For this specific transaction, including a fee greater than this number would cause it jump ahead of the zero-fee transactions in the mempool.
```

Comment:
> As for fee calculations, the minimum transaction fee is 5 mojos per cost (anything less than this will be treated as 0).

I don't think this is quite accurate. You can put any fee you'd like on your spend bundle, and if it makes it into the mempool it will be prioritized based on the fee/cost against other spend bundles. The significance of the 5 mojo/cost is that if the mempool is full, you need *at least* 5 mojos per cost in order to kick some other transaction our of the mempool (to make space for your spend bundle).

Reply:
OK, I didn't realize this detail. I updated it in the latest version.